### ISSUE TITLE
Issue with sqlite-net-pcl

### ISSUE THREAD

**Issue by DaFreeLum** [MicrosoftDocs/xamarin-docs#2311]
The latest version of the nuGet package sqlite-net-pcl v1.6.292 I downloaded caused a crash 
I worked out the only difference was the version of the package so when I rolled it back to sqlite-net-pcl v1.5.321and all was OK 

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5ce30880-c4f0-78f7-1ee0-6507e378201e
* Version Independent ID: a6ad2818-c0d2-e3ef-1262-e143d49cc348
* Content: [Store data in a local SQLite.NET database - Xamarin](https://docs.microsoft.com/en-gb/xamarin/get-started/quickstarts/database?pivots=windows)
* Content Source: [docs/get-started/quickstarts/database.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/get-started/quickstarts/database.md)
* Product: **xamarin**
* Technology: **xamarin-forms**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by davidbritch** [comment]
Related: https://github.com/MicrosoftDocs/xamarin-docs/issues/2310

**Comment by davidbritch** [comment]
Hi @DaFreeLum 

I'll take a look at this later this week. It'd be better if the tutorial worked with the latest sqlite-net-pcl NuGet package.

**Comment by davidbritch** [comment]
Hi @DaFreeLum 

I loaded the sample and updated the sqlite-net-pcl NuGet package to the latest version. Like you, I then experienced a crash when I ran the app. However, after quitting Visual Studio, deleting the bin and obj folders throughout the solution, and reloading it into Visual Studio, the app worked fine with the updated NuGet.

I've updated the sample the doc links to, to use the latest version of sqlite-net-pcl.
